lib/rucio/core/rule.py

```diff
@@ -2424,12 +2891,14 @@ def get_evaluation_backlog(expiration_time: int = 600, *, session: "Session") ->
 
     cached_backlog = REGION.get('rule_evaluation_backlog', expiration_time=expiration_time)
     if cached_backlog is NO_VALUE:
-        stmt = select(func.count(models.UpdatedDID.created_at), func.min(models.UpdatedDID.created_at))
+        stmt = select(
+            func.count(models.UpdatedDID.created_at),
+            func.min(models.UpdatedDID.created_at)
+        )
         result = session.execute(stmt).scalars().one()
         REGION.set('rule_evaluation_backlog', result)
-        return result
-    else:
-        return cached_backlog  # type: ignore
+        return result  # type: ignore
+    return cached_backlog  # type: ignore
```

Comment:
1. In this case, the type hint is correct. Can you try to use `result._tuple()` to [coerce the type checker](https://docs.sqlalchemy.org/en/20/core/connections.html#sqlalchemy.engine.Row._tuple)?

 2. Maybe you need to use the following for the cached value?
    ```python
    cached_backlog: Union[NO_VALUE, tuple[int, datetime]] = REGION.get('rule_evaluation_backlog', expiration_time=expiration_time)
    ```

Reply:
You can fully ignore this function. It was addressed in #6552.